screens/Onboarding/styles.ts

```diff
@@ -1,88 +1,71 @@
-import { StyleSheet } from 'react-native' 
+import { StyleSheet } from "react-native"
 import {
   white,
   black,
   lightGray,
   lightPeach,
   peach,
   shadowColor,
-} from '../../assets/colors/colors' 
+} from "../../assets/colors/colors"
 
 import {
   BUTTON_HEIGHT,
   BUTTON_RADIUS,
   BUTTON_TEXT_FONT_SIZE,
-} from '../../assets/global/constants' 
+} from "../../assets/global/constants"
 
 const styles = StyleSheet.create({
-  border: {
-    flex: 1,
-    flexDirection: 'column',
-    justifyContent: 'space-between',
-  },
   button: {
-    alignItems: 'center',
+    alignItems: "center",
     backgroundColor: lightPeach,
     borderColor: peach,
     borderRadius: BUTTON_RADIUS,
     borderWidth: 2,
     elevation: 8,
     height: BUTTON_HEIGHT,
-    justifyContent: 'center',
+    justifyContent: "center",
     marginBottom: 10,
     shadowColor,
     shadowOffset: { width: 0, height: 4 },
     shadowOpacity: 0.3,
     shadowRadius: 5,
-    width: '100%',
+    width: "100%",
   },
   buttonGoogle: {
-    alignItems: 'center',
     backgroundColor: white,
     borderRadius: BUTTON_RADIUS,
     borderWidth: 1,
-    display: 'flex',
-    flexDirection: 'row',
+    display: "flex",
+    flexDirection: "row",
     height: BUTTON_HEIGHT,
-    justifyContent: 'flex-start',
+    justifyContent: "flex-start",
     paddingLeft: 15,
     shadowColor,
     shadowOffset: { width: 0, height: 4 },
     shadowOpacity: 0.3,
     shadowRadius: 5,
-    width: '100%',
+    width: "100%",
   },
 
-  buttonPlaceholder: {
-    alignItems: 'center',
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'flex-start',
-  },
   buttonText: {
     color: black,
     fontSize: BUTTON_TEXT_FONT_SIZE,
   },
-  buttonTextGoogle: {
-    color: black,
-    fontSize: BUTTON_TEXT_FONT_SIZE,
-    fontWeight: 'bold',
-  },
+
   container: {
-    alignItems: 'center',
+    alignItems: "center",
     flex: 1,
     gap: 1,
     padding: 20,
   },
-  footer: {
-    display: 'flex',
-    flexDirection: 'row',
-    position: 'absolute',
 
+  footer: {
+    flex: 1,
+    flexDirection: "row",
+    position: "absolute",
+    zIndex: -1,
```

Comment:
Why did you need a `zIndex: -1` here? Wasn't `bottom: insets.bottom` not enough for the footer to be anchored to the bottom?

Reply:
The footer was anchored at the bottom without the zIndex: -1 indeed. The problem was when opening the keyboard, the footer came up. So I solved it this way. Do you think it would be better to do as with the other screens, with a useEffect?